lib/lib.js

```diff
@@ -74,7 +74,12 @@ function maybeTestForPlusOne(config, comment) {
   return /\+1/.test(comment);
 }
 
+exports.formatReviewedByMetaData = formatReviewedByMetaData;
 function formatReviewedByMetaData(reviewer) {
+  if (!reviewer.email) {
```

Comment:
I usually find having one exit point (one `return` statement) easier to follow when reading code, but that's very subjective. It's not always desirable too, so it's not a hard rule in my book.

As for avoiding the duplicated format strings, I find it help prevents bugs where, in a later change, the name format string would be updated to another format string in one code path but not the other. I think it's a bit more objective than the previous point about having a single return statement. In general, I try to avoid duplication in code unless it makes things significantly more complicated.